boomerang/src/lib.rs

```diff
@@ -57,24 +72,24 @@ fn check_canister_ids() {
     );
     assert_eq!(
         Principal::from_text("buwm7-7yaaa-aaaar-qagva-cai").unwrap(),
-        NICP_LEDGER_ID
+        Principal::from_slice(&[0, 0, 0, 0, 2, 48, 1, 170, 1, 1])
     );
     assert_eq!(
         Principal::from_text("jcmow-hyaaa-aaaaq-aadlq-cai").unwrap(),
-        WTN_LEDGER_ID
+        Principal::from_slice(&[0, 0, 0, 0, 2, 0, 0, 215, 1, 1])
     );
     assert_eq!(
         Principal::from_text("tsbvt-pyaaa-aaaar-qafva-cai").unwrap(),
-        WATER_NEURON_ID
+        Principal::from_slice(&[0, 0, 0, 0, 2, 48, 1, 106, 1, 1])
     );
 }
 
-#[cfg(target = "wasm32-unknown-unkown")]
+#[cfg(target_arch = "wasm32")]
```

Comment:
why?

Reply:
The previous configuration was not working for me so I got inspired by the cfg provided in the water_neuron directory